decidim-budgets/app/cells/decidim/budgets/budgets_list/show.erb

```diff
@@ -7,7 +7,7 @@
 <% end %>
 
 <% if budgets.where.not(id: voted.map(&:id)).exists? %>
-  <div id="budgets" class="card__list-list">
+  <div id="budgets" class="card__list-list !space-y-5">
```

Comment:
Request for change
I can't see any reference in tailwind of a class starting with a bang !. can link me the doc? This is quiet unsual

Reply:
https://v3.tailwindcss.com/docs/configuration#important-modifier :wink: